utils/src/treev2/quickPickWizard/ContextValueQuickPickStep.ts

```diff
@@ -6,14 +6,21 @@
 import { TreeItem } from 'vscode';
 import * as types from '../../../index';
 import { parseContextValue } from '../../utils/contextUtils';
+import { PickFilter } from './common/PickFilter';
 import { GenericQuickPickOptions, GenericQuickPickStep } from './GenericQuickPickStep';
 
 export interface ContextValueFilterQuickPickOptions extends GenericQuickPickOptions {
     contextValueFilter: types.ContextValueFilter;
 }
 
 export class ContextValueQuickPickStep<TContext extends types.QuickPickWizardContext, TOptions extends ContextValueFilterQuickPickOptions> extends GenericQuickPickStep<TContext, TOptions> {
-    protected override isDirectPick(node: TreeItem): boolean {
+    readonly pickFilter: PickFilter = new ContextValuePickFilter(this.pickOptions);

```

Comment:
Since this isn't being called within the constructor, I'm not 100% sure that `this.pickOptions` will be available. I don't know the order for sure but I thought that class-scoped initializers like this get called _before_ the constructor.